test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -0,0 +1,128 @@
+package kubernetes
```

Comment:
We will be porting the other orchestration engines which may or may not have upstream e2e tests.